docs/setup-caddy-proxy-with-minio.md

```diff
@@ -27,20 +33,28 @@ proxy / localhost:9000 {
 }
 ```
 
-## 4. Recipe Steps
-
-### Step 1: Start `minio` server.
+### 3.2 Change the IP Addresses
+Change the DNS entry and IP address/port in the configuration file according to your Minio and DNS configuration, and save the file. The example above defaults to `your.public.com` and `localhost:9000`, respectively.
```

Comment:
Should omit reference to "DNS entry" as this would be confusing (implies DNS change is needed for this to succeed), needs to be worded to imply publicly accessible hostname